<?xml version="1.0"?>
<launch>
  <group ns="main_robot">
    <!-- NODES -->
    <node pkg="tf" type="static_transform_publisher" name="laser_broadcaster" args="0 0 0 0 0 0 world table 1000" />
    <node pkg="eurobot_sim" name="tf_broadcaster" type="tf_broadcaster.py"  output="screen"/>
    <node pkg="rviz" name="rviz_laserscan" type="rviz" args="-d $(find eurobot_sim)/config/laserscan.rviz" output="screen"/>

    <!-- PARAMS -->
    <!-- robot -->
    <param name="robot_name" type="string" value="main_robot" />
    <param name="robot_description" command="cat $(find eurobot_sim)/models/field.urdf" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" respawn="false"/>

    <!-- robot size -->
    <param name="dim_x" type="double" value="420" />
    <param name="dim_y" type="double" value="190" />
    <param name="dim_z" type="double" value="350" />

    <!-- initial coordinates -->
    <param name="start_x" type="double" value="293.0" />
    <param name="start_y" type="double" value="425.0" />
    <param name="start_a" type="double" value="4.71" />

  </group>

</launch>
